packages/@glimmer/runtime/test/ember-component-new-test.ts

```diff
@@ -0,0 +1,113 @@
+import { module, RenderTests, test, classes } from "@glimmer/test-helpers";
```

Comment:
Should probably rename this to something other than `ember-component-new-test`? Maybe `ember-component-new-final-FINAL-2`?